lua/nvim-treesitter/parsers.lua

```diff
@@ -2184,6 +2184,16 @@ list.vala = {
   maintainers = { "@Prince781" },
 }
 
+list.vento = {
+  install_info = {
+    url = "https://github.com/ventojs/tree-sitter-vento",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
```suggestion
```
Not needed if that is the default branch. You _do_ need to add an entry to the lockfile, though, to have CI run.